pkgs/tools/wayland/wleave/default.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, at-spi2-atk
+, pkg-config
+, glib
+, gtk3
+, gtk-layer-shell
+, installShellFiles
+, scdoc
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wleave";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "AMNatty";
+    repo = pname;
+    rev = "${version}";
```

```suggestion
    rev = version;
```
> Unnecessary string conversions should be avoided.
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax